test-scenarios/fastboot3-test.ts

```diff
@@ -22,7 +22,7 @@ function upgradeFastbootFormat() {
   let upgrader = new Project('fastboot_upgrader', {
     files: {
       'index.js': `
-        const Funnel = require('broccoli-funnel');
+        const { Funnel } = require('broccoli-funnel');
```

Comment:
broccoli-funnel was not pinned, rather it was a random transitive dependency that was being pulled in. The latest version changed it's API, and decoupled its "functional" api from its "classical" api. (thanks @rwjblue)

To correct this, I explicitly added broccoli-funnel as a devDep + imported it correctly